addons/point_of_sale/models/product_combo.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ProductCombo(models.Model):
+    _name = "product.combo"
+    _description = "Product combo choices"
+
+    name = fields.Char(string="Name", required=True)
+    combo_product_ids = fields.One2many("product.combo.item", "combo_id", string="Products in Combo")
```

Comment:
```suggestion
    combo_line_ids = fields.One2many("product.combo.item", "combo_id", string="Products in Combo")
```
odoo developers are so used to "lines" instead of "items". Let's just use "line".